server/odc-service/src/main/java/com/oceanbase/odc/service/session/ConnectConsoleService.java

```diff
@@ -339,7 +339,7 @@ public AsyncExecuteResultResp getMoreResults(@NotNull String sessionId, String r
         int timeout = Objects.isNull(timeoutSeconds) ? DEFAULT_GET_RESULT_TIMEOUT_SECONDS : timeoutSeconds;
         boolean shouldRemoveContext = context.isFinished();
         try {
-            List<JdbcGeneralResult> resultList = context.getMoreSqlExecutionResults(timeout);
+            List<JdbcGeneralResult> resultList = context.getMoreSqlExecutionResults(timeout * 1000);
```

Comment:
we may rename the variable to show the unit of this time, suggestion: gettingResultTimeoutSeconds